wasatch/applog.py

```diff
@@ -123,6 +124,17 @@ def __init__(self,
         if self.logfile is not None:
             set_location(self.logfile)
 
+        # append file size limits are enforced upon program restart
+        if append_arg.lower() == "true":
+            # limit to 300mb if --log-append is explicitly set to "True"
+            append = 300*1024*1024
```

Comment:
This would limit logfiles to 300MB with no way for the user to override it.  I literally just generated and used a 10GB logfile this morning :-(  There needs to be a way for users to disable this behavior (while still appending).